adobe/aem-core-cif-components
# CIF-1060 Implement the log in process (SignIn component)

## Description

Implement the sign-in component as part of the "account management" topic. 

The account management process is handled by components written in React. Those components are rendered in the current navigation panel and they interact with it using DOM events.

### Steps to verify:

Prerequisites: configure dispatcher proxy to point to your Magento instance; make sure you have a valid customer.

1. Install the content package.
2. Go to any page and open the navigation panel
3. Click the "Sign in" button at the bottom of the panel.
4. Enter the credentials of the customer. 

You should see the "My Account" section in the panel. There are currently two entries there - a non-functional to a "Purchase history" page and the "Sign out" link.

<img width="395" alt="Screenshot 2019-10-03 13 32 16" src="https://user-images.githubusercontent.com/1407821/66119865-3cfb9100-e5e2-11e9-9480-de60032fb5b4.png">

In addition to that: when the user is signed in the "Sign in" button should be replaced by the user information
<img width="393" alt="Screenshot 2019-10-03 13 30 13" src="https://user-images.githubusercontent.com/1407821/66119799-189fb480-e5e2-11e9-8126-df54f6635b81.png">

## Related Issue





CIF-1060

## Motivation and Context


Support for registered shoppers in the demo store-front.

## How Has This Been Tested?





Functional tests / unit tests

## Screenshots (if appropriate):

## Types of changes




## Checklist:
